app/src/main/res/layout/fragment_patient_chart.xml

```diff
@@ -9,7 +9,7 @@
     OR CONDITIONS OF ANY KIND, either express or implied.  See the License for
     specific language governing permissions and limitations under the License.
 -->
-<RelativeLayout
+<android.support.v4.widget.DrawerLayout
```

Comment:
I think you need to put a `RelativeLayout` as the first child in the `DrawerLayout`, like this:

```xml
<DrawerLayout>
  <RelativeLayout>
    <!-- Content goes here -->
  </RelativeLayout>
  <ListView id="@+id/chart_list" / >
</DrawerLayout>
```

The `DrawerLayout` doesn't document that it works like a `RelativeLayout`, for all the content views you add, so we shouldn't assume that in the layout here. In fact, I'm surprised that the content renders correctly with this layout, because `DrawerLayout` inherits from `ViewGroup`, not from `RelativeLayout`.